DAS/README.md

```diff
@@ -9,7 +9,7 @@ These steps assume that you are in the data-acquisition-system root directory an
 1. `cd DAS`
 2. `poetry install`
 3. `poetry shell`
-4. Run any DAS command, _e.g._ `python -m das.V3_mqtt_recorder`
+4. Run any DAS command, _e.g._ `python -m das.V3_mqtt_recorder -v`
```

Comment:
-m stands for module. It allows us to run `das.V3_mqtt_recorder` which is inside a folder. The fancy definition is that it allows us to _execute python code from the command line via modulename rather than filename_.